### DISCUSSION TITLE
Does "monitoring changes" mean LiveSync?

### DISCUSSION THREAD

**Discussion by xieve** [vrtmrz/obsidian-livesync#282] [Q&A]
When configuring hidden files and customization sync, many options refer to a "monitoring changes" option, that I cannot find. Is the LiveSync mode what is meant? If so, please change the corresponding options to say "[…] if LiveSync is enabled" instead of "[…] if monitoring changes is enabled".

**Comment by vrtmrz** [comment]
Thank you for pointing this out! 
`Monitoring changes` is now in the `Hatch` as `Scan changes on customization sync`. Description labels are outdated (Really sorry).
I thought that we sometimes need this toggle disabled immediately, due to using internal API. However, fortunately, the internal API is recently stable and I moved that into the `Hatch`.
Now, I think that we are able to remove that note from the descriptions. I will address it in the next release!
Thank you so much!
